tests/testserver.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import threading
+import requests
+
+from tests.testserver.server import Server, consume_socket_content
```

Comment:
I'm confused how this import is working right now. Maybe it's too late for me to be reviewing code?

Reply:
😬 this is me running `git add tests` and not paying attention to garbage in the directory. Disregard this file.

Reply:
I await your force-push to remove it 😄

Reply:
Ok, we should be updated now.